lib/src/main/java/graphql/nadel/engine/util/GraphQLUtil.kt

```diff
@@ -365,7 +367,18 @@ fun ExecutionIdProvider.provide(executionInput: ExecutionInput): ExecutionId {
     return provide(executionInput.query, executionInput.operationName, executionInput.context)
 }
 
-fun ServiceExecutionResult.copy(
+fun NadelIncrementalServiceExecutionResult.copy(
+    data: MutableJsonMap = this.data,
+    errors: MutableList<MutableJsonMap> = this.errors,
+    extensions: MutableJsonMap = this.extensions,
+    incremental: List<IncrementalPayload>? = this.incremental,
+    incrementalItemPublisher: Publisher<DelayedIncrementalPartialResult> = this.incrementalItemPublisher,
+    hasNext: Boolean = this.hasNext,
+): ServiceExecutionResult {
+    return newIncrementalServiceExecutionResult(data, errors, extensions, incremental, incrementalItemPublisher, hasNext)
+}
```

Comment:
Should just make `NadelIncrementalServiceExecutionResult` a `data` class and remove this.